napalm_junos/utils/junos_views.yml

```diff
@@ -200,8 +200,7 @@ junos_lldp_neighbors_detail_view:
     parent_interface: {lldp-local-parent-interface-name: unicode}
     remote_port: {lldp-remote-port-id: unicode}
     remote_chassis_id: {lldp-remote-chassis-id: unicode}
-    remote_port: {lldp-remote-port-id: unicode}
```

Comment:
Maybe I was not able to express myself properly: please do not remove this one - the line that has to be removed is in ```junos.py``` (will add a note)